release/nightly_tests/dataset/sort.py

```diff
@@ -119,6 +119,7 @@ def make_block(count: int, num_columns: int) -> Block:
         else:
             ds = ds.sort(key="c_0")
         ds.fully_executed()
+        print(ds.stats())
```

Comment:
nit: to keep the stats coming after the "finished" etc, assign `ds.stats()` to a variable and use later for print.

Reply:
@jianoaix - updated.